recipes-kernel/linux/linux-yocto_%.bbappend

```diff
@@ -3,7 +3,9 @@ FILESEXTRAPATHS_prepend := "${THISDIR}:"
 SRC_URI += "file://meson64-kmeta;type=kmeta;destsuffix=meson64-kmeta"
 
 COMPATIBLE_MACHINE_khadas-vim3 = "khadas-vim3"
+COMPATIBLE_MACHINE_khadas-vim3-sdboot = "khadas-vim3-sdboot"
 KMACHINE_khadas-vim3 = "meson-gx"
+KMACHINE_khadas-vim3-sdboot = "meson-gx"
```

Comment:
These as been fixes on https://github.com/superna9999/meta-meson/pull/96/files

I juste merged them, can you rebase over ?